### PR TITLE
Add missing mapages

### PR DESCRIPTION
**Pull request by andypugh**
Address Issue #3025

### CODE REVIEW

**Comment by rene-dev** [comment]
@petterreinholdtsen can you make sure they get converted to asciidoc for master?

**Comment by petterreinholdtsen** [comment]
[Rene Hopf]
> @petterreinholdtsen can you make sure they get converted to asciidoc
> for master?

OK.  I will have a look.

-- 
Happy hacking
Petter Reinholdtsen

File: docs/man/man1/qtplasmac-materials.1 [hunk 1]
```diff
@@ -36,7 +36,7 @@ The file can be created by text entry or by importing text from a CAM tool file.
 .SH "SEE ALSO"
 See the QtPlasmaC section of the LinuxCNC Documentation for more information.
 
-http://linuxcnc.org/docs/2.9/html/plasma/qtplasmac.html#_material_converter
+http://linuxcnc.org/docs/stable/html/plasma/qtplasmac.html#_material_converter
```
Comment (petterreinholdtsen) [inline docs/man/man1/qtplasmac-materials.1:39]: Including this change in the commit to merge forward to master fill cause a merge conflict, as the file has been removed in master.  You might want to consider handling it in a different commit.

File: docs/man/man1/melfagui.1 [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+.SH DESCRIPTION
+\fBmelfagui\fR is one of the sample \fBVismach\fR GUIs for LinuxCNC,
+simulating a Mitsubishi serial manipulator
+
+See the main LinuxCNC documentation for more details.
+
+http://linuxcnc.org/docs/html/gui/vismach.html
+.
+.SH "SEE ALSO"
+\fBLinuxCNC(1)\fR
```
Comment (petterreinholdtsen) [inline docs/man/man1/melfagui.1:38]: The manual page file name is 'linuxcnc' with lower case letters, so I recommend changing these references to linuxcnc(1).

**Comment by petterreinholdtsen** [comment]
Once the change is in master, I will run the scripts/man2adoc-migrate
script to convert the groff files to adoc files.

-- 
Happy hacking
Petter Reinholdtsen

File: docs/man/man1/qtplasmac-materials.1 (hunk 1 above)
Comment (andypugh) [reply docs/man/man1/qtplasmac-materials.1:39]: I will handle that as merge conflict.

**Pull request merged**